### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by leilenatekle-onshape**
README updated to reflect the new "add new extension" feature. Update also includes minor changes to clarify where users have to make custom entries.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -52,7 +52,7 @@ Make changes to code at two places for the new URL that Heroku has produced, as
        ........
        "repository": {
        "type": "git",
-       "url": "https://newURL-from-heroku.herokuapp.com/"
+       "url": "https://<newURL-from-heroku>.herokuapp.com/“
```
Comment (awk) [inline README.md:55]: It looks like this is the wrong quotation mark character at the end of the line. Perhaps a setting in your editor? Or just a typo? I don't see the problem on any of the other lines you changed.

**Review by awk — CHANGES_REQUESTED** [review]
Fix the quotation character issue and this looks good to :shipit:

File: README.md (hunk 1 above)
Comment (leilenatekle-onshape) [reply README.md]: Yes! Just a typo. Fixed.

**Review by awk — APPROVED** [review]
:shipit:

**Pull request merged**